maintenance/health/servicehealthcheck/README.md

```diff
@@ -1,7 +1,24 @@
-# Health Checks for Services 
-* Makes it possible to have a health check for different services (e.g. postgres, redis)
+# Health Check
+* Makes it possible to add a health check for a service (e.g. postgres) to the list of checks checked when for `/health` and `/health/check`
 
-* A service has to implement the `HealthCheck` interface and register the health check with `servicehealthchecker.RegisterHealthCheck(hc HealthCheck, name string)`
+* `/health` returns OK(200) if all registered and required health checks are healthy and ERR(503) if any health check is not healthy
```

Comment:
(optional) indicate that we are talking about http status codes and not a literal response body with `OK(200)` as content. Ignore this comment if the actual contents of the response are just that.